script.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
```

Comment:
Whats the use of "use strict" ?

Reply:
"use strict" is to indicate that the code should be executed in "strict mode" which is useful to execute coding with less errors.